.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "ush/python/wxflow"]
```

Comment:
While I'm not opposed to submodules, why does wxflow get a submodule when every other external package gets handled by `checkout.sh`?

Reply:
Because, I don't think we need to wait for `checkout.sh`, `build_all.sh` and then finally `link_workflow.sh` to be able to setup and create an experiment or use for testing.  
In the short/medium term, we will fold `wxflow` in the `gfs_workflow` environment at that time, this submodule can be removed.